aptos-move/aptos-replay-benchmark/src/overrides.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! Defines different overrides for on-chain state used for benchmarking. With overrides, past
+//! transactions can be replayed on top of a modified state, and we can evaluate how it impacts
+//! performance or other things.
+
+use aptos_logger::error;
+use aptos_types::{
+    on_chain_config::{FeatureFlag, Features, OnChainConfig},
+    state_store::{state_key::StateKey, state_value::StateValue, StateView},
+};
+use serde::Serialize;
+use std::collections::HashMap;
+
+/// Stores feature flags to enable/disable, essentially overriding on-chain state.
+pub struct OverrideConfig {
+    enable_features: Vec<FeatureFlag>,
```

Comment:
maybe enabled_ and disabled_

Reply:
I think additional_enabled_ ... even? Because those add on top whatever is on the state